### ISSUE TITLE
Table 6 :  Multi-Stage

### ISSUE THREAD

**Issue by monchana** [microsoft/SoftTeacher#143]
Is there a code where I can reference the Multi-Stage experiment conducted on Table 6?
It looks like the experiment itself is mentioned in issue [#11](https://github.com/microsoft/SoftTeacher/issues/11#issuecomment-918984107)

**Comment by MendelXu** [comment]
You can just train a supervised model on the labeled data and then retrain a model using both labeled data and unlabeled data.

**Comment by monchana** [comment]
Thank you for your reply,

I have question regarding your reply.

If I train a supervised model on labeled data (which is same as baseline) and retrain a model using both types of data, 
isn't it simply inserting a trained model into a E2E method? How is it same as Multi-Stage of Table 6?

**Comment by MendelXu** [comment]
`Multi-Stage` here means you have to do semi-supervised learning in `train a model on labeled data`->`get label on unlabeled data`->`train on both labeled data and unlabeled data`.

**Comment by monchana** [comment]
If both stages of training are conducted with same training code, do they have same data augmentation? From your paper, it seems strong augmentation is used to train student. (While teacher infers weakly augmented images)

**Comment by MendelXu** [comment]
No, you'd better refer to the paper of STAC. In the multi-stage setting, the same setting is used. ( strong augmentation is used to train student While teacher infers weakly augmented images)

**Comment by monchana** [comment]
Thank you for your reply!
